### ISSUE TITLE
Prisma Dependency 

### ISSUE THREAD

**Issue by agubler** [kleydon/prisma-session-store#60]
It doesn't look like the prisma dependency is used by the session store as a runtime dependency, would it be possible to move this to a development dependency? At the moment this is inflating the size of the node_modules for a production installation as the prisma dependency and binaries is installed twice (due to differing versions)

**Comment by simontaisne** [comment]
It's even more problematic now that Prisma 3 is available. When running `prisma -v` with `prisma-session-store` installed this is what I get:

```
prisma                : 2.30.3
@prisma/client        : 3.6.0
```

**Comment by kleydon** [comment]
Fixed for v3.1.1
